network/drivers/none.md

```diff
@@ -0,0 +1,38 @@
+---
+title: None network driver
+description: How to disable networking by using the none driver
+keywords: network, none, standalone
+redirect_from:
+  - /network/none/
+---
+
+If you want to completely disable the networking stack on a container, you can
```

Comment:
From @thediveo (see [here](https://github.com/docker/docs/issues/10415#issuecomment-1570929224)):

> Thanks for the pointer so I could take a sneak preview. It looks really great, especially with the updated filtering/firewall notes.
>
> The only nitpicking I still feel to vent here is with the first sentence on network none:
>
> > If you want to completely disable the networking stack on a container [...]
> 
> I find the "disable" choice highly unfortunate, also based on feedback from colleagues. May I recommend a tiny rephrasing as "If you want to completely isolate the networking stack on a container [...]"?

I think the suggestion is quite great and better communicate what is really happening.